pages/api/api.tsx

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+const BASE_URL = 'https://panda-market-api.vercel.app';
+
+export async function getArticles(params = {}) {
+    const allowedParams = ["orderBy", "pageSize", "page", "keyword"];
+    const invalidParams = Object.keys(params)
+        .filter(key => !allowedParams.includes(key));
+        
+    if (invalidParams.length > 0) {
+        console.error(`Invalid parameters detected: ${invalidParams.join(", ")}`);
+    }
+
+    try {
+        const response = await axios.get(`${BASE_URL}/articles`, {
```

Comment:
P3:
어떤 값이 내려올지 타입을 명시해주시면 더 좋을 것 같습니다.
```suggestion
        const response = await axios.get<ArticleProps[]>(`${BASE_URL}/articles`, {
```